plugins/nanospell/plugin.js

```diff
@@ -190,7 +190,7 @@
 			// reached the end of block,
 			// so just return what we've walked
 			// of the current word.
-			if (word) return { word: word, range: wordRange };
+			return { word: word, range: wordRange };
```

Comment:
This makes sense.  I guess the other option would be that we could bump the offset up before returning.